server/src/graphql/queries/app.ts

```diff
@@ -6,6 +6,11 @@ export const app: QueryResolvers['app'] = async (_, { appId }, { userId }) => {
     throw new Error('Unauthorized');
   }
 
-  const app = await prisma.app.findOne({ where: { id: appId } });
+  const app = await prisma.app.findOne({
```

Comment:
I think this should rather be in a resolver. That way it's not executed if you don't ask for the subfields :)